### PR TITLE
feat: guild scores

### PR DESCRIPTION
**Pull request by FrancescoBorzi**

### CODE REVIEW

File: functions.php [hunk 1]
```diff
@@ -40,14 +40,11 @@ function getPlayerName($guid)
   return $row['name'];
 }
 
-function getPlayerColor($guid)
+function getPlayerColorByRace($race)
```
Comment (Helias) [inline functions.php]: return type and input param type

File: functions.php [hunk 2]
```diff
@@ -74,6 +71,15 @@ function getPlayerColor($guid)
   return $color;
 }
 
+function getPlayerColorByGuid($guid)
```
Comment (Helias) [inline functions.php]: return type and input param type

File: guild.php [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+                    characters.gender AS character_gender,
+                    characters.level AS character_level,
+                    characters.totalKills AS character_totalKills
+                    FROM pvpstats_players 
+                    INNER JOIN pvpstats_battlegrounds ON pvpstats_players.battleground_id = pvpstats_battlegrounds.id 
+                    INNER JOIN characters ON pvpstats_players.character_guid = characters.guid
+                    INNER JOIN guild_member ON guild_member.guid = characters.guid 
+                    WHERE characters.deleteDate IS NULL 
+                    AND pvpstats_players.winner = 1 
+                    AND guild_member.guildid = %d %s LIMIT 0,100",
```
Comment (Helias) [inline guild.php]: these conditions should be written before to save more CPU resources.
```sql
INNER JOIN pvpstats_battlegrounds ON pvpstats_players.battleground_id = pvpstats_battlegrounds.id 
INNER JOIN characters ON pvpstats_players.character_guid = characters.guid AND characters.deleteDate IS NULL 
INNER JOIN guild_member ON guild_member.guid = characters.guid AND guild_member.guildid = %d
```

File: guild.php [hunk 4]
```diff
@@ -0,0 +1,150 @@
...
+        $guildId,
+        $players_group_and_order);
+
+    $result = $db->query($query);
+
+    if (!$result)
+        die("Error querying: " . $query);
```
Comment (Helias) [inline guild.php]: bracket

File: guild.php [hunk 5]
```diff
@@ -0,0 +1,150 @@
...
+
+    $position = 0;
+    $prev_score = 0;
+
+    while (($row = $result->fetch_array()) != null)
+    {
+        if ($prev_score != $row['count'])
+            $position++;
```
Comment (Helias) [inline guild.php]: bracket

File: guild.php [hunk 6]
```diff
@@ -0,0 +1,150 @@
...
+                  <th class="text-center">#</th>
+                  <th class="text-center">Character</th>
+                  <th class="text-center">&#9679;</th>
+                  <th class="text-center">Level</th>
+                  <th class="text-center">Kills</th>
+                  <th class="text-center">Victories</th>
+                </tr>
+              </thead>
+              <tbody>
+                <?php getGuildPlayers(3); ?>
```
Comment (Helias) [inline guild.php]: ?

Comment (Helias) [reply guild.php]: is it not better to return the characters and print them using `<?= getGuildPlayers(3) ?>`

File: guild.php [hunk 7]
```diff
@@ -0,0 +1,150 @@
...
+              </tbody>
+            </table>
+          </div>
+        </div>
+
+      </div>
+
+      <div id="footer">
+        <hr>
+        <?php /* I worked hard to make PvPstats open and free for everyone. Please, do not remove the credits. */ ?>
```
Comment (Helias) [inline guild.php:138]: we could convert this into HTML comment to keep it also for client-side

File: guild.php (hunk 6 above)
Comment (FrancescoBorzi) [reply guild.php]: yes, also I should remove the `3`

File: guild.php [hunk 8]
```diff
@@ -0,0 +1,150 @@
...
+            <table class="table table-striped">
+              <thead>
+                <tr>
+                  <th class="text-center">#</th>
+                  <th class="text-center">Character</th>
+                  <th class="text-center">&#9679;</th>
+                  <th class="text-center">Level</th>
+                  <th class="text-center">Kills</th>
+                  <th class="text-center">Victories</th>
+                </tr>
```
Comment (Helias) [inline guild.php:125]: if you put text-center on tr you could remove the other ones probably

File: guild.php (hunk 7 above)
Comment (FrancescoBorzi) [reply guild.php:138]: yes, but in another PR

**Review by Helias — APPROVED** [review]

**Pull request merged**